doc/navigating.rst

```diff
@@ -5,7 +5,7 @@ Using the Synchronization Client
 .. index:: navigating, usage
 
 The ownCloud Desktop Client remains in the background and is visible as an icon 
-in the system tray (Windows, KDE), status bar (Mac OS X), or notification area 
+in the system tray (Windows, KDE), menu bar (Mac OS X), or notification area
```

Comment:
Now that we're dealing with correctness. Quoting https://en.wikipedia.org/wiki/Macintosh_operating_systems:

> The current Mac operating system is macOS, originally named "Mac OS X" until 2012 and then "OS X" until 2016.